samples/minimum_criteria.rst

```diff
@@ -0,0 +1,20 @@
+.. _definition_and_minimum_criteria:
+
+Definition & Minimum Criteria
+#############################
+
+A sample is an application in Zephyr that briefly showcases how to use a feature, subsystem, or domain.
+
+===========================================
+Minimum Requirements/ Criteria for Samples:
+===========================================
+
+* Samples must not use Zephyr's testing framework.
+* Leveraged in CI for quality confidence, its primary purpose is to exemplify the usage of subsystem, feature, or domain.
+
+  * *Note: Must not serve the purpose of a testcase.*
```

Comment:
Thats not how we do notes in RST, and I think this should be a main bullet and not a note.